File: redcap-api/python/delete_records.py
#!/usr/bin/env python
from config import config
import pycurl, cStringIO, json

buf = cStringIO.StringIO()

data = {
    'token': config['api_token'],
    'action': 'delete',
    'content': 'record',
    'records[0]': '1'
}
ch = pycurl.Curl()
ch.setopt(ch.URL, config['api_url'])
ch.setopt(ch.HTTPPOST, data.items())
ch.setopt(ch.WRITEFUNCTION, buf.write)
ch.perform()
ch.close()
print buf.getvalue()
buf.close()

File: redcap-api/perl/delete_records.pl
#!/usr/bin/env perl
use strict;
use warnings;
use LWP::Curl;

my %config = do 'config.pl';
my $data = {
    token => $config{api_token},
    action => 'delete',
    content => 'record',
    'records[0]' => '1'
};
my $ch = LWP::Curl->new();
my $content = $ch->post(
    $config{api_url},
    $data,
    $config{referer}
);
print $content;